lib/kinesis_client.rb

```diff
@@ -75,16 +77,8 @@ def push_batch(batch)
       records: batch.to_a,
       stream_name: @stream_name
     })
-
-    if resp.failed_record_count > 0 
-      failure_message = {
-        failures: resp.failed_record_count,
-        failures_data: filter_failures(resp)
-      }
-      $logger.warn("Batch sent to #{config[:stream_name]} with failures: #{failure_message}")
-    else
-      $logger.info("Batch sent to #{config[:stream_name]} successfully")
```

Comment:
I'd like to keep these logging statements if possible! (or something close to them). Also agree with Daniel that anything that is being output with `puts` should be formally logged. (For many reasons, one of which being that we are looking at new instrumentation tools that can hypothetically pull monitoring data out of the structured logging output).

Reply:
yea the puts was just leftover from debugging tests :+)